src/SprykerEco/Zed/Easycredit/Persistence/Mapper/EasycreditPersistenceMapper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Easycredit\Persistence\Mapper;
+
+use Generated\Shared\Transfer\PaymentEasycreditApiLogTransfer;
+use Generated\Shared\Transfer\PaymentEasycreditOrderIdentifierTransfer;
+use Orm\Zed\Easycredit\Persistence\SpyPaymentEasycreditApiLog;
+use Orm\Zed\Easycredit\Persistence\SpyPaymentEasycreditOrderIdentifier;
+
+class EasycreditPersistenceMapper implements EasycreditPersistenceMapperInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\PaymentEasycreditApiLogTransfer $easycreditApiLogTransfer
+     * @param \Orm\Zed\Easycredit\Persistence\SpyPaymentEasycreditApiLog $spyPaymentEasycreditApiLog
+     *
+     * @return \Orm\Zed\Easycredit\Persistence\SpyPaymentEasycreditApiLog
+     */
+    public function mapPaymentEasycreditApiLogTransferToEntity(
+        PaymentEasycreditApiLogTransfer $easycreditApiLogTransfer,
+        SpyPaymentEasycreditApiLog $spyPaymentEasycreditApiLog
```

Comment:
`$paymentEasycreditApiLogEntity`
And please fix it in other places for entities